openshift/deploy.sh

```diff
@@ -16,7 +16,7 @@ fi
 
 [ \( "${THOTH_LOCAL}" \) ] && oc cluster up --routing-suffix="${THOTH_ROUTING_SUFFIX}"
 oc new-project thoth-frontend
-oc new-project thoth-middleend
+oc new-project ${THOTH_MIDDLEEND_NAMESPACE}
 oc new-project thoth-backend
```

Comment:
The conf.sh file is sourced above. As there are configured THOTH_*_NAMESPACE parameters, I would suggest to use them here - if the given namespace already exists, the script will simply print error message and continue with execution.